### DISCUSSION TITLE
Multi-selection in dashboard

### DISCUSSION THREAD

**Discussion by Gunkev** [medusajs/medusa#3748] [Feature Requests]
Hi, I would like to be able to select and perform an action at once in the Medusa dashboard. For example, given the product page, discount or pricing page, It just feels kind of annoying to delete an item one by one. It would be nice to have a multi-selector so that the user can select a group of particular items and click on the delete button to delete or disable it.

**Comment by olivermrbl** [comment]
This is a great suggestion @Gunkev and actually already on our radar. 

We have much of the functionality in place - both API and UI component - so I'd be happy to accept a PR, if you are up for it :)

**Reply by Gunkev** [reply]
Sure @olivermrbl would be a great experience once again in open source. Do I need any additional resources beforehand?

**Comment by redareda9** [comment]
Still non-existent in 2024. 
We are not able to delete multiple products in the Product List Dashboard.

**Comment by Hermitauge** [comment]
this would be the best thing that you guys could implement right now before the stable version of v2, i don't have the skill but I would highly recommend this tweaked version of the shadcn table, just need Vic locked in 48hours playing lego with the code  =)) : https://github.com/sadmann7/shadcn-table

You guys are doing fantastic.

**Reply by Kushal2024** [reply]
Correct. Multi select option should be available.

**Comment by kishanio** [comment]
Came here looking for multiselect and later realizing that its tricky to integrate shadcn or similar UI frameworks unless one ejects medusa admin dashboard from the starter pack setup?
